cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -110,20 +110,13 @@ message Partition {
 }
 
 /**
-Specifies the volume "shape"
+VolumeDef is a simple, versioned, flexible format for describing which grid points of a seismic volume contain data.
 
-The json data describes a map from an inline value to all of its valid crosslines. The valid crosslines can
-be defined by a combination of value lists, or ranges (defining a start and end crossline and a step value).
+The grid is described per line. Each line contains a series of descriptors that specify the points on the axis perpendicular to the line. 
+For example, a volumedef that is indexed as inline-major will use inline numbers as keys in the lines object, and have descriptors with crossline numbers.
 
-Example:
+For more information refer to [VolumeDef schema](https://cognite-seismic-sdk.readthedocs-hosted.com/en/latest/volumedef.html#volumedef-overview "SDK documentation")
```

Comment:
I tested this locally, and this does not work. It seems like protoc-gen-docs doesn't support links, so I think the best we can do is a raw URL.